assets/js/Containers/FrevaGPT/index.js

```diff
@@ -0,0 +1,438 @@
+  }
+
+  async handleStop() {
+    // stop of thread only possible if a thread id is given
+    if (this.props.frevaGPT.thread) {
+      await fetch(
+        `/api/chatbot/stop?` +
+          new URLSearchParams({
+            auth_key: process.env.BOT_AUTH_KEY,
+            thread_id: this.props.frevaGPT.thread,
+          }).toString()
+      );
+    }
+
+    // abort fetch request anyway (especially if no thread is given)
+    // if (abortController.current) abortController.current.abort();
+
+    this.setState({ loading: false });
+    this.props.dispatch(
+      addElement({ variant: "UserStop", content: "Request stopped manually" })
+    );
+  }
+
+  toggleBotSelect() {
+    this.setState({ hideBotModelList: !this.state.hideBotModelList });
+  }
+
+  renderAlert() {
+    return (
+      <Alert key="botError" variant="danger">
+        The bot is currently not available. Please retry later.
+      </Alert>
+    );
+  }
+
+  renderBotContent() {
+    return (
+      <>
+        <Col md={4}>
+          <SidePanel />
+        </Col>
+
+        <Col md={8}>
+          {this.state.showSuggestions ? (
+            <Row className="mb-2 g-2">
+              {botSuggestions.map((element) => {
+                return (
+                  <div key={element} className="col-md-3">
+                    <OverlayTrigger
+                      key={element}
+                      overlay={<Tooltip>{element}</Tooltip>}
+                    >
+                      <Button
+                        className="h-100 w-100"
+                        variant="outline-secondary"
+                        onClick={() => this.handleSubmit(element)}
+                      >
+                        {truncate(element)}
+                      </Button>
+                    </OverlayTrigger>
+                  </div>
+                );
+              })}
+            </Row>
+          ) : null}
+
+          <ChatBlock />
+
+          <PendingAnswerComponent
+            content={this.state.dynamicAnswer}
+            variant={this.state.dynamicVariant}
+          />
+
+          {this.state.loading && !this.state.dynamicAnswer ? (
+            <Row className="mb-3">
+              <Col md={1}>
+                <Spinner />
+              </Col>
+            </Row>
+          ) : null}
+
+          <Row>
+            <Col md={12}>
+              <InputGroup className="mb-2 pb-2">
+                <FormControl
+                  type="text"
+                  value={this.state.userInput}
+                  onChange={this.handleUserInput}
+                  onKeyDown={this.handleKeyDown}
+                  placeholder="Ask a question"
+                />
+                {this.state.loading ? (
+                  <Button variant="outline-danger" onClick={this.handleStop}>
+                    <i className="bi bi-stop-fill"></i>
+                  </Button>
+                ) : (
+                  <Button variant="outline-success" onClick={this.handleSubmit}>
```

Comment:
You have to call `this.submitUserInput` instead of `handleSubmit`, otherwise you are passing an Event to `this.handleSubmit` instead of a string